cwl/src/main/scala/cwl/MyriadOutputTypeToWomValue.scala

```diff
@@ -27,15 +29,12 @@ object MyriadOutputTypeToWomValue extends Poly1 {
     _.fold(MyriadOutputInnerTypeToWomValue)
   }
 
-  // Evaluate all the types and make a WomArray from it
+  // TODO: Not sure what the right thing to do is here, for now go over the list of types and use the first evaluation that yields sueccess
```

Comment:
This entire file is a mystery to me, but could you comment on what's the situation where you'd get more than one success here?